package.json

```diff
@@ -19,7 +19,7 @@
     "axios": "^1.1.2",
     "file-loader": "^6.2.0",
     "immer": "^9.0.15",
-    "keycloak-js": "^19.0.1",
+    "keycloak-js": "^20.0.2",
```

Comment:
Hold den til samme versjon av keycloak vi kjører i sso, som er 19.0.2

Reply:
https://github.com/Informasjonsforvaltning/sso/blob/main/Dockerfile#L24

Reply:
@NilsOveTen Will do! Vet du om dependabot kommer til å åpne ny PR med 20.0.2 igjen? Og hvis ja, går det an å be den til å ignorere denne versjonen? Eller evt oppdatere keycloak i SSO? Hva er best?

Reply:
Det går an å si til dependabot at den skal ignorere dette, ja. Men ser ikke noen grunn til det, vi bør nok heller oppgradere keycloak versjonen i sso

Reply:
OK. Da endrer jeg den til 19.0.2 og lager issue i forbindelse med keycloak oppgradering. 👍